pkgs/by-name/na/nanomq/package.nix

```diff
@@ -62,24 +62,24 @@ in stdenv.mkDerivation (finalAttrs: {
   buildInputs = [ cyclonedds libmysqlclient mariadb mbedtls sqlite zeromq ];
 
   cmakeFlags = [
-    "-DBUILD_BENCH=ON"
-    "-DBUILD_DDS_PROXY=ON"
-    "-DBUILD_NANOMQ_CLI=ON"
-    "-DBUILD_ZMQ_GATEWAY=ON"
-    "-DENABLE_RULE_ENGINE=ON"
-    "-DNNG_ENABLE_SQLITE=ON"
-    "-DNNG_ENABLE_TLS=ON"
+    (lib.cmakeBool "BUILD_BENCH" true)
+    (lib.cmakeBool "BUILD_DDS_PROXY" true)
+    (lib.cmakeBool "BUILD_NANOMQ_CLI" true)
+    (lib.cmakeBool "BUILD_ZMQ_GATEWAY" true)
+    (lib.cmakeBool "ENABLE_RULE_ENGINE" true)
+    (lib.cmakeBool "NNG_ENABLE_SQLITE" true)
+    (lib.cmakeBool "NNG_ENABLE_TLS" true)
   ];
 
-  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-return-type -Wno-implicit-function-declaration -Wno-error=implicit-int";
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=int-conversion";
 
   # disabled by default - not 100% reliable and making nanomq depend on
   # mosquitto would annoy people
   doInstallCheck = false;
   nativeInstallCheckInputs = [
     mosquitto
     netcat-gnu
-    (python3.withPackages (ps: with ps; [ jinja2 requests paho-mqtt ]))
```

Comment:
I think it should be added back:

```
> Running phase: installCheckPhase
> Traceback (most recent call last):
>   File "/build/source/.github/scripts/test.py", line 13, in <module>
>     from ws_test import ws_test
>   File "/build/source/.github/scripts/ws_test.py", line 3, in <module>
>     import paho.mqtt.client as mqtt
> ModuleNotFoundError: No module named 'paho'
```

Reply:
I guess it will at least stop people going down the avenue of thinking "oh I just need to give it paho..."

(not sure I meant to commit this come to think...)